From 39ba8053e3249a5564fda64090a98080e1224413 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 7 May 2019 19:56:26 +0200
Subject: [PATCH 3/3] [DNM] Squash of lineage-16.0-android-9.0.0_r37

Squashed commit of the following:

commit fff27838050900ba7f4a9ed3c6e4adff9c9cb6cc
Merge: 7288a72ef b7419efc9
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Feb 20 04:17:23 2019 +0000

    Snap for 5322077 from b7419efc90ace578cdf0318f57717d3ffd65c800 to pi-qpr3-release

    Change-Id: Ia050f8e1db17ae18986968aac2d833dd2fa4877e

commit b7419efc90ace578cdf0318f57717d3ffd65c800
Author: Bill Yi <byi@google.com>
Date:   Sat Feb 16 08:27:49 2019 -0800

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: Ic5060be6572556efa7b8a8add48b2e002b06e2c3

Change-Id: Ifc306250bfafe6edc843bca32f90d64616f1e9d1
---
 res/values-ca/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values-ca/strings.xml b/res/values-ca/strings.xml
index 692a15cf8..82d8bee3b 100644
--- a/res/values-ca/strings.xml
+++ b/res/values-ca/strings.xml
@@ -74,7 +74,7 @@
     <string name="no_storage" product="default" msgid="5922012950780548098">"Per utilitzar la càmera, insereix una targeta SD."</string>
     <string name="preparing_sd" product="default" msgid="7435693655017362767">"S\'està preparant la targeta SD…"</string>
     <string name="access_sd_fail" product="default" msgid="2634861611082499811">"No s\'ha pogut accedir a la targeta SD."</string>
-    <string name="time_lapse_title" msgid="3267978566401228497">"Gravació a intervals"</string>
+    <string name="time_lapse_title" msgid="3267978566401228497">"Gravació en mode de filmació a intervals"</string>
     <string name="feedback_description_camera_access" msgid="419090951070170132">"L\'aplicació no s\'ha pogut connectar a la càmera."</string>
     <string name="feedback_description_save_photo" msgid="8658767358989083228">"La foto o el vídeo no s\'han desat al dispositiu."</string>
     <string name="capturing" msgid="5255164204641920893">"S\'està capturant."</string>
-- 
2.17.1

